presto-docs/src/main/sphinx/optimizer/history-based-optimization.rst

```diff
@@ -0,0 +1,97 @@
+joinNodeStatistics              Include total number of input rows and number of rows with NULL join key for both probe and build input         RandomizeNullKeyInOuterJoin
+tableWriterNodeStatistics       Number of tasks in table writer                                                                                 ScaledWriterRule
+partialAggregationStatistics    Row count and size of input and output of the partial aggregation node                                          PushPartialAggregationThroughExchange
+=============================== =============================================================================================================== ===========================================================================================================
+
+How to use HBO?
+---------------
+
+Presto supports using historical statistics in query optimization. In HBO, statistics of current query are stored and can be used in optimizations of future queries.
+The Redis HBO Provider can be used as storage for the historical statistics. HBO is controlled by the following session properties:
+
+=========================================================== =========================================================================================================================================================================================================== ===============
+Session property                                            Description                                                                                                                                                                                                 Default value
+=========================================================== =========================================================================================================================================================================================================== ===============
+use_history_based_plan_statistics                           Enable using historical statistics for query optimization                                                                                                                                                   False
+track_history_based_plan_statistics                         Enable recording the statistics of the current query as history statistics so as to be used by future queries                                                                                               False
+history_based_optimizer_timeout_limit                       Timeout for history based optimizer                                                                                                                                                                         10 seconds
+restrict_history_based_optimization_to_complex_query        Enable history based optimization only for complex queries, i.e. queries with join and aggregation      True
```

Comment:
Please fix the formatting for the default value `True`. See screenshot of local build.
<img width="969" alt="Screenshot 2024-03-04 at 10 06 41 AM" src="https://github.com/prestodb/presto/assets/7013443/64e5cb3b-c40a-4918-9787-c04d867711f4">